### DISCUSSION TITLE
`assetPrefix` returns 404 for` _next/*` files.

### DISCUSSION THREAD

**Discussion by fr-nevin** [vercel/next.js#16922] [Help] [answered]
## Goal

**Serve `_next/static` and other `_next/*` files with `assetPrefix:'${country}` which is being passed from `.env` file.**

##Package.json

```
"build": "next build",
"analyze": "ANALYZE=true yarn build",
"start": "next start",
"dev": "yarn clean && concurrently \"yarn dev:pc\" \"yarn dev:sp\" -n \"pc,sp\" -c \"green.dim,yellow.dim\"",
"dev:pc": "UA=pc next dev -p 3000",
"dev:sp": "UA=sp next dev -p 3002",
"debug": "NODE_OPTIONS='--inspect' next dev",
```
## Background
I am migrating from using express custom server to **not using one.** 
**Nextjs Version**: 9.5.2
**Local server**: `yarn dev` OR `yarn build && yarn start`

## Problem
For **images**, it works fine
```
 <Image
    src={`/${country}/static/images/download-1.png`}
    alt={"download-1"}
/>
```

But, on the network call for `_next` files, I **get 404.**

![image](https://user-images.githubusercontent.com/45579756/92449826-92c6b900-f1f5-11ea-9b7e-9a2d2ed00692.png)

**Doesn't work =>** `http://dev.gu-global.com/jp/_next/static/chunks/pages/_app.js?ts=1599552331624`
**Works ==>** `http://dev.gu-global.com/_next/static/chunks/pages/_app.js?ts=1599552331624`

Here, If I remove the country(jp), it works. (I do like to point out that with an nginx configuration, I can make it work. But I would like not to use nginx for this.)

## Question
1) If I add assetPrefix, its supposed to add the prefix and fetch the file without further configurations, right?

Could someone help with this? Or if you need more info, lemme know.
cc: @timneutkens

**Comment by fr-nevin** [comment, marked as answer]
I fixed it using redirect/rewrite methods on Nextjs next.config.js. Cheers!

**Reply by amilcruise** [reply]
hi @fr-nevin , would you be able to share here what you did? Tried using rewrites/redirects but still couldn't make it work. Appreciate your help 🙏
